pkg/cover/html.go

```diff
@@ -103,6 +105,184 @@ func (rg *ReportGenerator) DoHTML(w io.Writer, progs []Prog) error {
 	return coverTemplate.Execute(w, d)
 }
 
+var csvFilesHeader = []string{
+	"Filename",
+	"CoveredLines",
+	"TotalLines",
+	"CoveredPCs",
+	"TotalPCs",
+	"TotalFunctions",
+	"CoveredPCsInFunctions",
+	"TotalPCsInFunctions",
+}
+
+func (rg *ReportGenerator) convertToStats(progs []Prog) ([][]string, error) {
```

Comment:
This should return a slice of structures, not []string.
It very hard to follow any of the code that later uses data[0], or data[3]. It's unclear what are these. You also unnecessary convert to strings and then back. That's usually a flavor of wrong design. This should return typed data:
```
type something struct {
  file string
  cover int
  ...
}
```